content/en/docs/started/installing-kubeflow.md

```diff
@@ -334,7 +334,7 @@ The following table lists distributions which are <em>maintained</em> by their r
 ### Kubeflow Manifests
 
 The Kubeflow Manifests are aggregated by the Manifests Working Group and are intended to be
-used as the **base of packaged distributions**.
+used by users with Kubernetes knowledge and as the base of packaged distributions.
```

Comment:
I think this language is verging on violating the [charter of the manifests working group](https://github.com/kubeflow/community/blob/master/wg-manifests/charter.md) which explicitly prevents the creation of an "official distribution" of Kubeflow. It must be clear that the primary purpose of the manifests is to be the base of distributions, and `"users with Kubernetes knowledge"` really sounds like everyone.

Here is my suggestion that gives less of an implication that the manifests are the "official distribution":

```suggestion
used as the base of _packaged distributions_, and by those who want to build a custom deployment.
```

Reply:
It is not verging on that.

To cite from the charter:

"In scope
**Code**, Binaries and Services
Maintain tooling to automate copying manifests from upstream app repos.
Maintain a catalog that will **allow users to install Kubeflow apps and common services easily on Kubernetes**, either on the cloud or on-prem, without depending on external cloud services or closed source solutions. Those manifests are deployed using kubectl and kustomize and include: ..."

So i think 

```
The Kubeflow Manifests are aggregated by the Manifests Working Group and are intended to be
used by users with Kubernetes knowledge and as the base of packaged distributions.
```

is still correct.

CC @kubeflow/kubeflow-steering-committee